test/specs/commonTests.js

```diff
@@ -304,10 +304,18 @@ export const isConformant = (Component, options = {}) => {
           'You may need to hoist your event handlers up to the root element.\n'
         )
 
+        let expectedArgs = [eventShape]
+        let errorMessage = 'was not called with (event)'
+
+        if (_.has(Component.propTypes, listenerName)) {
+          expectedArgs = [eventShape, props]
```

Comment:
`props` here isn't the Component's full props object, it's just:
```
const props = {
  ...requiredProps,
  [listenerName]: handlerSpy,
  'data-simulate-event-here': true,
}
```

So we're really just testing that arbitrary props get passed through (e.g. that `data-simulate-event-here` prop). Tests for additional data, like `checked` of a `Checkbox`, should be handled in that Component's test file.